tests/test_dbapi.py

```diff
@@ -458,3 +458,34 @@ def test_get_all_assertions(api, initialize_list_assertions, begin, limit, expec
         first_batch = list(api.get_all_assertions(begin=0, limit=expected_count))
         if len(first_batch) == expected_count:
             assert assertions[0].id != first_batch[0].id
+
+
+def test_delete_authorization_success(api, dummy_badge_id, dummy_person_id):
+    """Test successful deletion of an authorization"""
+    result = api.add_authorization(dummy_badge_id, "test@tester.com")
+    assert result == ("test@tester.com", dummy_badge_id)
+    assert api.authorization_exists(dummy_badge_id, "test@tester.com") is True
+
+    result = api.delete_authorization(dummy_badge_id, "test@tester.com")
+    assert result == ("test@tester.com", dummy_badge_id)
+    assert api.authorization_exists(dummy_badge_id, "test@tester.com") is False
```

Comment:
In theory I would recommend to not use other parts of the api too much in the tests, because if these other parts fail in the future they will mess up your test. Instead I would recommend to do database queries directly for insertion and verification.